### PR TITLE
[PM-15180] - update copy for free trial banner

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-15180

## 📔 Objective

This PR updates the translation keys for the different trial banners to all have complete sentences to allow for proper translation.

## 📸 Screenshots

Uploading Screen Recording 2024-11-25 at 3.43.04 PM.mov…

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by djsmith85 — CHANGES_REQUESTED** [review]
@jaasen-livefront If you are changing the context/meaning of a message, you'll need to delete the old key+message and any usages and create a new pair.

This is to ensure that any messages that were translated with the old meaning are deleted and the new text is present for the translations

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -3870,17 +3870,17 @@
...
+    "message": "Your free trial ends in $COUNT$ days.",
     "placeholders": {
       "count": {
         "content": "$1",
         "example": "You must set up 2FA on your user account before you can join this organization."
       }
     }
   },
-  "freeTrialEndPromptAboveTwoDays": {
-    "message": "$ORGANIZATION$, your free trial ends in $COUNT$ days. To maintain your subscription,",
```
Comment (cyprain-okeke) [inline apps/web/src/locales/en/messages.json:3883]: The business requested that `To maintain your subscription,` be removed

Comment (jaasen-livefront) [reply apps/web/src/locales/en/messages.json:3883]: @cyprain-okeke Given how different languages break up sentences in different ways and the fact this translation in particular was broken between 2 keys it was decided to make both keys complete sentences in order to allow for proper translation.

Comment (cyprain-okeke) [reply apps/web/src/locales/en/messages.json:3883]: Please provide a video recording

Comment (jaasen-livefront) [reply apps/web/src/locales/en/messages.json:3883]: @cyprain-okeke Sorry I'm not sure what you mean. What would you like the recording of?

Comment (cyprain-okeke) [reply apps/web/src/locales/en/messages.json:3883]: run the application and test it, while testing record it and attached it to the pr.

Comment (jaasen-livefront) [reply apps/web/src/locales/en/messages.json:3883]: @cyprain-okeke Screen recording attached

File: apps/web/src/locales/en/messages.json [hunk 2]
```diff
@@ -3870,17 +3870,17 @@
...
     "message": "You are using an unsupported web browser. The web vault may not function properly."
   },
-  "freeTrialEndPrompt": {
-    "message": "Your free trial ends in $COUNT$ days. To maintain your subscription,",
+  "freeTrialEndPromptCount": {
+    "message": "Your free trial ends in $COUNT$ days.",
     "placeholders": {
       "count": {
         "content": "$1",
         "example": "You must set up 2FA on your user account before you can join this organization."
```
Comment (mzieniukbw) [inline apps/web/src/locales/en/messages.json:3878]: i just noticed, this example text is wrong.

File: apps/web/src/app/billing/shared/payment-method.component.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
     class="tw-cursor-pointer"
     rel="noreferrer noopener"
   >
-    {{ "routeToPaymentMethodTrigger" | i18n }}
+    {{ "routeToPaymentMethodTriggerV2" | i18n }}
```
Comment (mzieniukbw) [inline apps/web/src/app/billing/shared/payment-method.component.html:17]: Could not find this in `messages.json`

File: apps/web/src/locales/en/messages.json [hunk 4]
```diff
@@ -3870,17 +3870,17 @@
...
       }
     }
   },
-  "freeTrialEndPromptAboveTwoDays": {
-    "message": "$ORGANIZATION$, your free trial ends in $COUNT$ days. To maintain your subscription,",
+  "freeTrialEndPromptMultipleDays": {
+    "message": "$ORGANIZATION$, your free trial ends in $COUNT$ days.",
     "placeholders": {
       "count": {
```
Comment (mzieniukbw) [inline apps/web/src/locales/en/messages.json:3885]: the example text is wrong for the two placeholders.

File: apps/web/src/locales/en/messages.json [hunk 5]
```diff
@@ -3892,32 +3892,32 @@
...
   },
-  "freeTrialEndingSoonWithoutOrgName": {
-    "message": "Your free trial ends today. To maintain your subscription,"
+  "freeTrialEndingTodayWithoutOrgName": {
+    "message": "Your free trial ends today."
   },
-  "routeToPaymentMethodTrigger": {
-    "message": "add a payment method."
+  "clickHereToAddPaymentMethod": {
+    "message": "Click here add a payment method."
```
Comment (mzieniukbw) [inline apps/web/src/locales/en/messages.json:3920]:
```suggestion
    "message": "Click here to add a payment method."
```

**Review by mzieniukbw — CHANGES_REQUESTED** [review]

**Comment by jaasen-livefront** [comment]
@mzieniukbw Thanks for your feedback. Ready for review again.

**Review by mzieniukbw — APPROVED** [review]

**Review by djsmith85 — APPROVED** [review]
Changes are looking good!

**Review by shane-melton — APPROVED** [review]

**Comment by jaasen-livefront** [comment]
@gbubemismith @cyprain-okeke Would you guys mind taking a look?

**Review by gbubemismith — APPROVED** [review]
Looks good!

**Review by jonashendrickx — APPROVED** [review]

**Review by cyprain-okeke — APPROVED** [review]

**Pull request merged**